custom_components/unifiprotect/__init__.py

```diff
@@ -119,13 +123,13 @@ def setup(hass, config):
     use_ssl = conf.get(CONF_SSL)
     minimum_score = conf.get(CONF_MIN_SCORE)
     scan_interval = conf[CONF_SCAN_INTERVAL]
+    session = async_get_clientsession(hass)
```

Comment:
This `asyncio.ClientSession` will be used to make async requests in the `UpvServer` class.